### PR TITLE
[WALLET + REACT] Add Cometh Connect

### PR DESCRIPTION
**Pull request by yoanslvy**
Hello guys, We are happy to incorporate Cometh Connect to thirdweb ! 

Cometh Connect is a smart wallet based on Gnosis Safe contracts that provides biometric signatures (passwordless, non custodial), gasless transactions, account recovery, etc.

We understand the significance of this feature to our users, and we have committed ourselves to its implementation to enhance the usability of thirdweb and Cometh Connect.

If you want to go deeper:

Cometh Connect Docs: https://docs.cometh.io/connect/cometh-connect/what-is-connect

### CODE REVIEW

**Comment by MananTank** [comment]
@yoanslvy Thanks for PR, We are excited to get this merged! 

I will test this shortly 

Don't worry about the failing tests - those are not related to your code change

**Comment by MananTank** [comment]
Hey @yoanslvy - I can't get an apiKey for testing :( 

image

File: .changeset/shiny-owls-sit.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+"@thirdweb-dev/react-core": minor
+"@thirdweb-dev/wallets": minor
+"@thirdweb-dev/react": minor
```
Comment (joaquim-verges) [inline .changeset/shiny-owls-sit.md:4]: these should be patch, not minor

**Comment by yoanslvy** [comment]
Hello @MananTank  - Thanks we are also eager to get this merged ! 

At the moment we are whitelisting emails to provide apiKeys for users. 
Could you give me an email that I could whitelist? I'll create a project on the dashboard for you guys.

**Comment by MananTank** [comment]
@yoanslvy Awesome, Can you whitelist the below emails: 

manan@thirdweb.com
eng@thirdweb.com

**Comment by yoanslvy** [comment]
Hello @MananTank , I just added you guys to the whitelist, you can go to [https://app.cometh.io](https://app.cometh.io/login?callbackUrl=/dashboard) to access you dashboard and apiKey !

**Comment by yoanslvy** [comment]
By the way, how can we add a Reference for Cometh Connect in your doc? I saw that your provide wallet reference here: https://portal.thirdweb.com/wallet/metamask

**Comment by MananTank** [comment]
@yoanslvy - You can create a PR in [thirdweb docs](https://github.com/thirdweb-dev/docs) to add docs for cometh connect. 

I tested it out and added a connection UI in Modal. 

I was only able to get the Gnosis chain working. I was not able to connect to Mumbai ( even though the apiKey was for Mumbai ). I get "This network is not supported" error. Is this expected?

**Comment by yoanslvy** [comment]
Hello @MananTank,  Thanks for the modal ! 

Perfect, I'll also add a PR to the [thirdweb docs](https://github.com/thirdweb-dev/docs) then.

Regarding your issue, did you fetch the latest version of the PR before doing your test ? It should work on all of the supported networks

**Comment by MananTank** [comment]
@yoanslvy Hey, while trying to connect to Mumbai I am now getting below error: 

image

So, I tried adding `localhost:8005` in the Allowed domains in cometh dashboard, but I cna't add it

image

**Comment by MananTank** [comment]
Update: I am able to connect the wallet. I don't see any issue with merging the PR.

We will merge this soon 🎉

**Pull request merged**